src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -706,7 +777,6 @@ private string FormatComment(string substitutedText)
                     Debug.Assert(numLines > 0);
                 }
 
-                Debug.Assert(TrimmedStringStartsWith(lines[0], "/**"));
```

Comment:
nit: Consider adding a comment to replace the assertion (we may have `/**` or a positional property case)